interfaces/automated-testing/scd/scd.yaml

```diff
@@ -270,19 +273,22 @@ components:
         conflict with a lower-priority operational intent.  The regulator
         specifies whether an operational intent may create a conflict with other
         operational intents of the same priority.
+
+        For U-Space UAS flight authorisation requests, priority is required
```

Comment:
I don't think we need this note here -- since uss_qualifier is the one that will be populating this field, we can decide how it is populated in the implementation.  The `Priority` field that is populated by the USS is the one described by the ASTM standard and which doesn't (can't) have this note.  Instead, we'll validate its value using a check in uss_qualifier if the testing configuration has specified the U-space profile.

Reply:
I see your point. It makes sense.

Reply:
I wanted to capture that information somewhere for completeness without referring to the implementation since I am planning to retire this repository: https://github.com/swiss-foca/Protocol. Where would you recommend to put it?